addons/test_discuss_full/tests/test_livechat_hr_holidays.py

```diff
@@ -48,7 +48,7 @@ def test_operator_limit_on_leave(self):
             }
         )
         self.env["discuss.channel"].create(
-            livechat_channel._get_livechat_discuss_channel_vals(anonymous_name="Visitor")
+            livechat_channel._get_livechat_discuss_channel_vals(agent=livechat_channel._get_operator())
```

Comment:
We should not be calling this method manually anywhere. This should be replaced by doing the actual rpc.